.github/workflows/ci.yml

```diff
@@ -128,6 +131,9 @@ jobs:
           # explicitly don't mention wgpu-hal so that --all-features don't apply to it
           cargo clippy --target ${{ matrix.target }} -p wgpu -p wgpu-core -p wgpu-info -p player --examples --tests --all-features -- -D warnings 
 
+          # build docs
+          RUSTDOCFLAGS='-Dwarnings' cargo doc --target ${{ matrix.target }} -p wgpu -p wgpu-core -p wgpu-info -p player -all-features --no-deps
```

Comment:
can we ask it to build docs for all crates (instead of listing them individually)?

Reply:
Oh, so the reason I'm doing this is because of wgpu-hal's `--all-features` not working, same reason as above. I need to also check wgpu-hal, so I'll add that.